src/lib/fetch.ts

```diff
@@ -65,9 +65,10 @@ export class CrossFetch implements FetchInstance, Interceptable {
     let body: unknown;
 
     if (
-      (headers['content-type'] &&
-        headers['content-type'].includes(JSON_CONTENT)) ||
-      parameters.headers?.['accept']?.includes(JSON_CONTENT)
+      headers['content-type'] &&
+      headers['content-type'].includes(JSON_CONTENT) //||
+      //This should be used when we don't have a response content-type
```

Comment:
Please make this a TODO for when security handlers get updated to make requests themselves, as we talked.